quesma/util/utils_test.go

```diff
@@ -821,3 +821,26 @@ func TestExtractInt64(t *testing.T) {
 		assert.False(t, success)
 	}
 }
+
+func TestFieldEncoding(t *testing.T) {
+	tests := []struct {
+		field string
+		want  string
+	}{
+		{"@timestamp", "@timestamp"},
+		{"timestamp", "timestamp"},
+		{"9field", "_9field"},
+		{"field9", "field9"},
+		{"field::", "field__"},
+		{"host.name", "host_name"},
```

Comment:
Nit: you could add a couple instances of "strange Unicode characters" cases here.

Reply:
ok, Will do that in next PR(s)